Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/cxf/CXFSAMLCallbackHandler.java

```diff
@@ -76,35 +76,39 @@ public CXFSAMLCallbackHandler(final HOKSAMLAssertionBuilder builder) {
      */
     @Override
     public void handle(final Callback[] callbacks) throws IOException, UnsupportedCallbackException {
+        accessor = PropertyAccessor.getInstance();
         LOG.trace("CXFSAMLCallbackHandler.handle begin");
         for (final Callback callback : callbacks) {
             if (callback instanceof SAMLCallback) {
 
                 try {
 
                     final Message message = getCurrentMessage();
-
                     final Object obj = message.get("assertion");
-
-                    AssertionType custAssertion = getCustAssertion(obj);
-
+                    AssertionType custAssertion;
```

Comment:
Any reasons to break into separate line instead of AssertionType custAssertion = getCustAssertion(obj);